app/backend/src/clj/teet/thk/thk_mapping.clj

```diff
@@ -68,6 +68,8 @@
    "4003" :activity.name/detailed-design ;; Pöhiprojekt
    "4004" :activity.name/land-acquisition ;; Maaost
    "4005" :activity.name/construction ;; Teostus
+   "4006" :activity.name/owners-supervision ;; OJV
+   "4009" :activity.name/road-safety-audit ;; Tee Ohutus Audit
```

Comment:
Should these be mapped straight to task.types since they are not activities in TEET?

Reply:
They are activities in .csv and this mapping for parsing csv, so no they should not be mapped to task.types as they are not task types in .csv